vclib/src/commonMain/kotlin/at/asitplus/wallet/lib/agent/HolderAgent.kt

```diff
@@ -292,6 +471,102 @@ class HolderAgent(
         }
         return Holder.CreatePresentationResult.Signed(jws.serialize())
     }
+}
+
+// in openid4vp, the claims to be presented are described using a JSONPath, so compiling this to a JsonElement seems reasonable
+fun SubjectCredentialStore.StoreEntry.toJsonElement(): JsonElement {
```

Comment:
maybe put that into a separate file, so that this file here doesn't get over 500 lines long (which is way too much)

Reply:
Moved to `vclib/.../data/CredentialToJsonConverter` in commit 3e4c8cd54334e7b7ed0bb56817ae64d8ed0d8d0d

After refactoring for comment https://github.com/a-sit-plus/kmm-vc-library/pull/53#discussion_r1593812217 it got way too big though, so I also moved the `create*Presentation-files` to their own class: `vclib/.../agent/VerifiablePresentationFactory`
I'm not so sure whether this makes the code too loose and hard to grasp though, and I'm also unsure about the location and the name of the new class